streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -285,7 +284,6 @@ public void suspend() {
         assertThat(task00.state(), is(Task.State.RUNNING));
 
         assertThrows(StreamsException.class, () -> taskManager.handleRevocation(taskId00Partitions));
-        // TODO K9113: is it safe for the task to still be RUNNING, or should we trap it in SUSPENDING or something?
```

Comment:
I think it is safe to still be RUNNING, since when this happens the thread would be notified and be closing (and hence closing all tasks).